pkg/proxy/iptables/proxier.go

```diff
@@ -373,6 +373,13 @@ var iptablesJumpChains = []iptablesJumpChain{
 	{utiliptables.TableFilter, kubeProxyFirewallChain, utiliptables.ChainForward, "kubernetes load balancer firewall", []string{"-m", "conntrack", "--ctstate", "NEW"}},
 	{utiliptables.TableNAT, kubeServicesChain, utiliptables.ChainOutput, "kubernetes service portals", nil},
 	{utiliptables.TableNAT, kubeServicesChain, utiliptables.ChainPrerouting, "kubernetes service portals", nil},
+}
+
+// Duplicates of chains created in pkg/kubelet/kubelet_network_linux.go; we create these
+// on startup but do not delete them in CleanupLeftovers.
+var iptablesKubeletJumpChains = []iptablesJumpChain{
```

Comment:
so this is a bugfix, isn't it?

Reply:
Technically, yes; right now, if you run `kube-proxy --proxy-mode=iptables --cleanup`, then it will delete a chain that kubelet "cares about", and kubelet will eventually recreate it. But the chain in question is one that kubelet doesn't actually _use_, so nothing will break.

(I wouldn't have bothered to fix this except that it seemed bad to have kube-proxy delete `KUBE-FIREWALL` on `--cleanup`, but then if I was fixing that, it seemed wrong to not fix `KUBE-POSTROUTING` too...)